web-ui/src/main/resources/catalog/locales/en-admin.json

```diff
@@ -772,6 +772,8 @@
     "system/inspire/atomDisabled": "Disabled",
     "system/inspire/enable-help": "Enable INSPIRE CSW (ie. language support and INSPIRE GetCapabilities document) and INSPIRE indexing. INSPIRE themes thesaurus MUST be installed to properly index themes and annexes. Note: It does not enable the INSPIRE editor view mode (see iso19139/layout/config-editor.xml).",
     "system/inspire/enableSearchPanel-help": "Enable search panel (Ext search only).",
+    "system/inspire/remotevalidation/url": "INSPIRE remote validation URL",
```

Comment:
I would add also a placeholder value for the field with the url of actual service, for now is in a sandbox, but can be updated later when the service is "official". In any case the users can setup the service locally also.

This should help the users with the url format. Placeholder: `http://inspire-sandbox.jrc.ec.europa.eu/etf-webapp/`